openmdao/drivers/pyoptsparse_driver.py

```diff
@@ -467,6 +470,9 @@ def run(self):
                 rels = relevant[path]
                 wrt = [v for v in indep_list if dv_meta[v]['source'] in rels]
 
+            if not wrt:
```

Comment:
So, this is a constraint that is not dependent on any design variables, and we just skip it. Seems like we should at least warn about it, since it might mean there is a problem in the model.

Reply:
added warning

Reply:
I think the message should be more explicit, like "... does not depend on any design variables, and was not added to the optimization."

Reply:
added